Sources/BrowserServicesKit/PrivacyConfig/PrivacyConfigurationManager.swift

```diff
@@ -87,6 +87,7 @@ public class PrivacyConfigurationManager: PrivacyConfigurationManaging {
                 data = embedded
             } else {
                 let jsonData = embeddedDataProvider.embeddedData
+                // swiftlint:disable:next force_try
```

Comment:
Doing my best to make no actual logic changes where possible - my rationale here is that this has been a force unwrap for a very long time, so it's safer to just disable the lint warning rather than mess with it.